README.md

```diff
@@ -0,0 +1,31 @@
+# Neopixel Christmas Lights
```

Comment:
Maybe we should add a part about how to change the row size on the tree for others to use.